portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +26,44 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
 
 
 /**
-  * Servlet that returns some example content. 
+  * Servlet that returns comments in database. 
   * Note: We first added the Gson library dependency to pom.xml.
   */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  int maxNumComments = 1;
```

Comment:
Why are you making this a class wide variable instead of just in doGet?

Reply:
I am thinking of having a default value. I think it is better to show at least one comment when nothing is imported from the number of comments

Reply:
Let's call this defaultMaxComments and case on 
request.getParameter(inputNumComment) instead.
Also make this constant private and final for clarity.